### ISSUE TITLE
Slime error on initialization

### ISSUE THREAD

**Issue by alolis** [ghoseb/dotemacs#5]
Hello again!

I had to [compile emacs24 from source](http://chrisperkins.blogspot.gr/2011/07/building-emacs-24.html) due to Ubuntu launchpad weirdness, so now i have version 24.2.50.1 and then cloned dotemacs to my .emacs.d folder.

Even though everything seems to work properly i do get the following in initialization

```
.emacs.d/packs/clojure/lib/slime/slime.el:Error: Don't know how to compile nil
```

Any ideas? Do I just ignore it?

I am also using [nRepl](https://github.com/kingtim/nrepl.el) by the way and not swank-clojure since it's no longer supported (in case it's related)

**Comment by alolis** [comment]
UPDATE: I've downgraded to emacs 24.1.1 and everything works fine so that might help to fix the issue :)

**Comment by ghoseb** [comment]
Yeah, there are some weird (but harmless) issues with the git tip. Released versions usually work fine.

Let me know if it breaks for you again.

nREPL support is kind of baked in... check out `packs/clojure/init.el` and `packs/clojure/config/nrepl.el`.

**Comment by firesofmay** [comment]
This should fix it : http://ivan.kanis.fr/compile-slime-with-emacs-bzr.html

**Comment by ghoseb** [comment]
This should be fixed in 87426ddb33. Thanks @firesofmay for the tip!
